src/util/util.js

```diff
@@ -139,7 +139,7 @@ export function polygonizeBounds(min: Point, max: Point, buffer: number = 0, clo
     const polygon = [minBuf, new Point(maxBuf.x, minBuf.y), maxBuf, new Point(minBuf.x, maxBuf.y)];
 
     if (close) {
-        polygon.push(minBuf);
+        polygon.push(minBuf.clone());
```

Comment:
What issue did the clone resolve?

Reply:
I spent a great deal of time hunting a bug that was caused by a one point in the polygon being a "shared reference". I don't think it makes sense to have such points as it doesn't allow in-place modifications to the points in the array. For example the `resample` function would not work.